packages/graphiql/package.json

```diff
@@ -28,28 +29,28 @@
     ]
   },
   "scripts": {
-    "build": "yarn build-clean && yarn build-js && yarn build-flow && yarn build-demo",
-    "build-clean": "rimraf 'graphiql.{css,*min.js,js}' esm dist",
-    "build-js": "bash ./resources/build.sh",
+    "analyze-bundle": "NODE_ENV=production CDN=1 ANALYZE=1 webpack -c resources/webpack.config.js -p",
+    "build": "yarn build-clean && yarn build-js && yarn build-flow",
+    "build-clean": "rimraf esm dist",
+    "build-js": "babel src --ignore __tests__ --out-dir dist/ && ESM=1 babel src --ignore __tests__ --out-dir esm/",
+    "build-bundles": "yarn build-bundles-clean && yarn build-bundles-dev && yarn build-bundles-min",
+    "build-bundles-dev": "NODE_ENV=development CDN=1 webpack --config resources/webpack.config.js -d",
+    "build-bundles-min": "NODE_ENV=production CDN=1 webpack --config resources/webpack.config.js -p",
```

Comment:
webpack --bail 

😅from past unfortunate experiences

Reply:
ah, thank you!